docs/content/get-started/set-up.md

```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
 <br/>
 
 
-## Clone the repos
+## Get the GDK and Starter Project Source Code
+
+### Zip Download
+
+If you are unfamiliar with Git, you can download a Zip file of the GDK and its Starter Project [here](https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3). 
+
+Once you have done that, you can skip the rest of this page and move on to [Open the FPS Starter Project]({{urlRoot}}/content/get-started/open-project.md).
```

Comment:
Mention that they need to unzip the file before moving on

And maybe make this bit bold so people don't download a zip _and_ clone the repos (if they truly have no idea what they're doing)

Reply:
perhaps put the second paragraph into an information callout